packages/chonky/src/components/file-list/ListEntry.tsx

```diff
@@ -38,37 +39,46 @@ export const ListEntry: React.FC<FileEntryProps> = React.memo(
         const ChonkyIcon = useContext(ChonkyIconContext);
         const fileEntryHtmlProps = useFileEntryHtmlProps(file);
         return (
-            <div className={classes.listFileEntry} {...fileEntryHtmlProps}>
-                <div className={commonClasses.focusIndicator}></div>
+            <div id="chonkyListFileEntry" className={classes.listFileEntry} {...fileEntryHtmlProps}>
+                <div id="chonkyFocusIndicator" className={commonClasses.focusIndicator}></div>
                 <div
+                    id="chonkySelectionIndicator"
                     className={c([
                         commonClasses.selectionIndicator,
                         classes.listFileEntrySelection,
                     ])}
                 ></div>
-                <div className={classes.listFileEntryIcon}>
+                <div id={file?.isDir ? "chonkyListDirEntryIcon" : "chonkyListFileEntryIcon"} className={classes.listFileEntryIcon}>
```

Comment:
can the file be a dir here? i see the class is classes.listFileEntryIcon always

Reply:
Yes, the file can be a dir.

Reply:
My confusion comes from how we went about overwriting the styles: we thought overwriting the classes would work, then we discovered the classes have a generated id appended, so we can't overwrite them. But now, this id is set to a conditional isDir, when before the class was always chonky-listFileEntryIcon-xx-yy ... why does the id here change based on isDir, if the class doesn't ?